### PR TITLE
Check vdc threebot service

### PR DESCRIPTION
**Pull request by MElborolossy**
### Description

Create a service to check vdc threebot if down deploy a new threebot and delete the old one 

### Related Issues

#3099

### Checklist

- [x] [Pre-commit hook is installed](https://github.com/threefoldtech/js-ng#pre-commit) to do formatting checks before committing code...etc
- [ ] Tests included
- [x] Build pass
- [x] Documentation
- [x] Code format and docstrings

### CODE REVIEW

File: jumpscale/packages/vdc/services/check_threebot.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    def __init__(self, interval=60 * 60, *args, **kwargs):
+        """Check that Threebot Container is UP and Redeploy if DOWN
+        """
+        super().__init__(interval, *args, **kwargs)
+
+    def job(self):
+        zos = get_zos()
+        threebot_workload_types = [WorkloadType.Container, WorkloadType.Subdomain]
+        j.logger.info("Check VDC threebot service: Check if Threebot containers in all VDCs is UP")
+        for vdc_name in j.sals.vdc.list_all():
```
Comment (xmonader) [inline jumpscale/packages/vdc/services/check_threebot.py:18]: you need to yield control between each vdc using gevent.sleep

File: jumpscale/packages/vdc/services/check_threebot.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
+from jumpscale.loader import j
+
+from jumpscale.sals.zos import get as get_zos
+from jumpscale.tools.servicemanager.servicemanager import BackgroundService
+from jumpscale.clients.explorer.models import WorkloadType
+
+
+class CheckThreebot(BackgroundService):
```
Comment (xmonader) [inline jumpscale/packages/vdc/services/check_threebot.py:8]: let's make that runs hourly

Comment (MElborolossy) [reply jumpscale/packages/vdc/services/check_threebot.py:8]: It already runs hourly, interval defined in line 9

File: jumpscale/packages/vdc/services/check_threebot.py (hunk 1 above)
Comment (MElborolossy) [reply jumpscale/packages/vdc/services/check_threebot.py:18]: Updated

**Review by xmonader — APPROVED** [review]

**Pull request merged**